spec/models/comfy/cms/page_spec.rb

```diff
@@ -32,6 +84,21 @@
       end
     end
 
+    context 'ignores suppressed articles' do
+      let(:animals_tag) { Tag.create value: 'animals' }
+      let!(:animals_article) { create :english_article, keywords: [animals_tag] }
+
+      it 'order by most matching tags' do
+        create :english_article, keywords: [animals_tag], suppress_from_links_recirculation: true
+        not_suppressed_match_article = create :english_article,
```

Comment:
Can we extract this two line to a let? Or just the first one that is a spec dependency